src/test/app.js

```diff
@@ -0,0 +1,77 @@
+/*eslint no-invalid-this: "off" */
+import { exists } from 'fs'
+import { join } from 'path'
+import rimraf from 'rimraf'
+import { spawn } from 'child_process'
+
+describe('new app', function() {
```

Comment:
Good stuff here! I think we need to sync on whitespace style between functions and blocks. Avoiding excessing scrolling is good, but I things begin to blur a bit for me without the whitespace. I looked over eslint for a rule on this, but closest I could find is no more than one blank line. Are you aware of any rules on this?